Rawdata server: add a quick startup script for development purpose.

FunkySayu opened the pull request.

Commit:
feat(rawdata-server) improve logging on Riot API crash

Commit:
feat(rawdata-server): support https for requests python module

Commit:
feat(rawdata-server): use HTTPS for Riot API communication, HTTP for tests

Commit:
feat(rawdata-server): Support startup of MongoDB server through Bazel

Commit:
feat(rawdata-server): Create configuration directory and add them to gitignore

Commit:
feat(rawdata-server): Adding a tool for deploying a development stack.

ArchangelX360 commented on third_party/python/riotwatcher/riotwatcher.py:
```diff
@@ -244,6 +244,10 @@ def request_available(self):
 class RiotWatcher:
+
+    # Used for testing purpose, disabling https make all routes testable.
+    enable_https = True
```
Is the comment saying the opposite of what the line does or did I get it wrong?

FunkySayu replied:
Used for testing purpose < i.e. do not modify this for production code
disabling https make all routes testable. < creating an HTTPS server for tests is not a funny thing to do :)
enable_https = True < required by the Riot API

Should I detail a little bit more the comment?

ArchangelX360 replied:
"Used for testing purpose < i.e. do not modify this for production code"
This i. e. is way clearer for me than the original sentence.

Commit:
feat(rawdata-server): Changing sample values due to pull/16 changes

Commit:
feat(rawdata-server): Fixing test flakiness on rawdata handler

See [1] for more informations.

  [1]: http://stackoverflow.com/questions/31745040/pythons-simplehttpserver-started-in-a-thread-wont-close-the-port/31746811#31746811

Commit:
feat(rawdata-server): Sanitizing BUILD files

FunkySayu commented:
Fixed.

Commit:
feat(rawdata-server): Improve comment

cassand reviewed (approved):
I need this
